src/users/users.service.ts

```diff
@@ -70,6 +71,18 @@ export class UsersService {
     return this.usersRepository.findOne({ email });
   }
 
+  async getPermissions(email: string): Promise<PermissionDto> {
+    const user = await this.usersRepository.findOne({ email });
+
+    if (user.isAdmin) {
+      return new PermissionDto({
+        isAdmin: user.isAdmin,
+      });
+    } else {
+      throw new ForbiddenException('Forbidden');
+    }
```

Comment:
This check is performed by your guard

```suggestion
      return new PermissionDto({
        isAdmin: user.isAdmin,
      });
```